include/xeus-cling/xmanager.hpp

```diff
@@ -123,9 +125,11 @@ namespace xcpp
             {
                 (*m_magic_line[magic_name])(line);
             }
-            catch (const cxxopts::OptionException& e)
-            {
-                std::cerr << "UsageError: " << e.what() << "\n";
+            catch (const std::runtime_error& e) {
+                std::cerr << e.what() << std::endl;
+            }
+            catch (const std::logic_error& e) {
+                std::cerr << e.what() << std::endl;
```

Comment:
Since `logic_error` and `runtime_error` inherit form `std::exception`, I think a single catch on `std::exception` should be enough